search/src/main/scala/weco/api/search/services/ElasticsearchRequestBuilder.scala

```diff
@@ -5,12 +5,14 @@ import com.sksamuel.elastic4s.requests.searches.sort.FieldSort
 import com.sksamuel.elastic4s.Index
 import com.sksamuel.elastic4s.requests.searches.queries.Query
 import com.sksamuel.elastic4s.ElasticDsl._
+import weco.api.search.elasticsearch.templateSearch.TemplateSearchRequest
 import weco.api.search.models.SearchOptions
 
 trait ElasticsearchRequestBuilder[S <: SearchOptions[_, _, _]] {
   val idSort: FieldSort
 
-  def request(searchOptions: S, index: Index): SearchRequest
+  def request(searchOptions: S,
```

Comment:
I'm using Either here to allow me to do this work in stages.  Image search and countWorkTypes both still use a search with the query defined programmatically.

Eventually, this will flip to just TemplateSearchRequest.

Reply:
Comment to this effect would be valuable